lang/java/compiler/src/test/java/org/apache/avro/compiler/specific/TestSpecificCompiler.java

```diff
@@ -344,59 +337,9 @@ public void testSettingOutputCharacterEncoding() throws Exception {
         new String(fileInDefaultEncoding), equalTo(new String(fileInDifferentEncoding, differentEncoding)));
   }
 
-  @Test
```

Comment:
There are several test cases deleted, are they covered somewhere else? So far I just took a vague look at this PR, so I might have more comments later, but the changes on this test instantly caught my eyes. I'm not sure if these cases give proper coverage, but would it provide value if we wouldn't drop them, but adjust the asserts to Java date/time API instead?

Reply:
Lets park this one after the 1.9 release.